client/src/hooks/useGetOfflineStore.ts

```diff
@@ -0,0 +1,25 @@
+import  { useEffect, useState } from "react";
+import { useApolloOfflineClient } from "react-offix-hooks";
+
+/**
+ * React hook to detect network changes
+ * and return current network state
+ *
+ * Usage: const isOnline = useNetworkStatus();
+ *
+ */
+export const useGetOfflineStore = () => {
```

Comment:
related to my previous comment: You could make use of the `client.queue` instead of `client.offlineStore.getOfflineData()`

Reply:
Sweet, will have a look at that now